OpenKeychain/src/main/java/org/sufficientlysecure/keychain/securitytoken/SecurityTokenHelper.java

```diff
@@ -206,7 +206,7 @@ public void connectToDevice(final Context ctx) throws IOException {
         mPw1ValidatedForDecrypt = false;
         mPw3Validated = false;
 
-        if (mOpenPgpCapabilities.isHasAESSM()) {
+        if (mOpenPgpCapabilities.isHasSCP11bSM()) {
```

Comment:
so before this change, this routine was triggered on values 1 and 2, now it is triggered only by 3. is this related to the bug you mentioned? can you say a bit more about this?